types/zalgo-promise/index.d.ts

```diff
@@ -0,0 +1,6 @@
+// Type definitions for zalgo-promise 1.0
+// Project: https://github.com/krakenjs/zalgo-promise#readme
+// Definitions by: Daniel Shuy <https://github.com/daniel-shuy>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export { ZalgoPromise } from './promise';
```

Comment:
I think it has to support UMD, so Global script usage:
https://github.com/krakenjs/zalgo-promise#global
https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#identifying-a-umd-library

Reply:
Oh right, thanks!

What about CommonJS? Is there a way to export for both CommonJS and ES (as a class module)?
```ts
// UMD
export as namespace ZalgoPromise;

// CommonJS
export = ZalgoPromise;    // fails with "An export assignment cannot be used in a module with other exported elements"

// ES
export class ZalgoPromise<R> {
```

Reply:
Nevermind, turns out the `README` is outdated. The CommonJS and UMD modules are imported as:
```html
<!-- UMD -->
<script src="zalgo-promise.js"></script>
<script>
    new ZalgoPromise.ZalgoPromise( ... );
</script>
```js
// CommonJS
var ZalgoPromise = require('zalgo-promise');

new ZalgoPromise.ZalgoPromise( ... );
```
This removes the need to exported it as a class module, therefore there is no issue exporting for UMD, CommonJS and ES.

I've updated the PR accordingly, thanks!